nabla/test/backstop.ts

```diff
@@ -118,17 +120,30 @@ export default async function (environment: TestSuiteEnvironment) {
       await backstop.addSwapPool(address(swapPool1), 0);
       await backstop.addSwapPool(address(swapPool2), 0);
 
+
+      await asset1.burn(tester, await asset1.balanceOf(tester));
```

Comment:
Why do we have to `burn` before we `mint`?

Reply:
If this is related to 

> Specifically cleaning token balances before execution of the tests and modifying time delay for swapExecution.

Maybe we should find a clearer way to do this? How about a teardown function, or we add cleanup-related code to the end of each test? Otherwise we should maybe at least add a comment that explains why we burn before we mint.

Reply:
Right, I see your point. Although we don't support 'teardown' functions yet, we could just try to execute them (if any) in a new `finally {}` block at the end of the try-catch [here](https://github.com/pendulum-chain/wasm-deploy/blob/ec682a08f06673e1db5b2f637e13c87d7447d482/src/commands/test.ts#L411). This way it will be executed in any case. 

I'm also fine with your existing implementation actually, the only thing that bothers me a bit is that the comment reasoning why we need to burn before minting is only included in one file. And I see why since copy-pasting it into any file is annoying. But with a teardown function for each test, it might be self-explanatory enough that we don't even need a comment in the first place. WDYT?

Reply:
Yes I agree that would be the better approach. Could we perhaps open a separate issue and wait for this one? Since it would be touching a separate part of the code logic not just the tests but also how they run.
In the meantime if you are okay with it I can just copy the comment to all the setups so we don't forget what is going on.

Reply:
Sounds good 👍 could you create that follow-up ticket?

Reply:
Sure!